File: test/fixtures/languages/sql/accepts_ef_core_parameterized_interpolation_only/Queries.cs
await db.Database.ExecuteSqlInterpolatedAsync(
    $"INSERT INTO Users (Id) VALUES ({userId})");
db.Database.ExecuteSqlInterpolated($"DELETE FROM Users WHERE Id = {userId}");
await db.Database.ExecuteSqlRawAsync(
    $"UPDATE Users SET Name = {userName} WHERE Id = {userId}");
var query = $"SELECT * FROM Users WHERE Id = {userId}";

File: test/fixtures/languages/sql/accepts_generated_parameter_placeholder_lists/queries.ts
const placeholders = ids.map(() => '?').join(',');
db.prepare(`SELECT * FROM users WHERE id IN (${placeholders})`).all(...ids);
db.prepare(`SELECT * FROM users WHERE id IN (${ids.map(() => '?').join(',')})`).all(...ids);
db.prepare(`SELECT * FROM users WHERE id = ${userId}`);
